ci/matrix.sh

```diff
@@ -59,13 +59,6 @@ elif [ "$BUILD_TARGET" = "linux64" ]; then
   export BITCOIN_CONFIG="--enable-zmq --enable-glibc-back-compat --enable-reduce-exports --enable-crash-hooks"
   export CPPFLAGS="-DDEBUG_LOCKORDER -DENABLE_DASH_DEBUG -DARENA_DEBUG"
   export PYZMQ=true
-elif [ "$BUILD_TARGET" = "linux64_cxx17" ]; then
-  export HOST=x86_64-unknown-linux-gnu
-  export DEP_OPTS="NO_UPNP=1 DEBUG=1"
-  export BITCOIN_CONFIG="--enable-zmq --enable-glibc-back-compat --enable-reduce-exports --enable-crash-hooks --enable-c++17 --enable-werror --with-sanitizers=undefined"
```

Comment:
Hmmmm, we shouldn't just do this b/c of `--enable-werror --with-sanitizers=undefined` which are not on linux64/32

Also the fact that this one doesn't do functional tests. We should probably keep this, but just rename it to be like linux64_nofun :D